### PR TITLE
bug: normalize channelids to JSON encodable entities

### LINKED ISSUE #691: CHID fails to encode to JSON for bridged connections
https://sentry.prod.mozaws.net/operations/autopush-stage/issues/356530/

Check CHID and UAID for encoding issues before sending out data.

### PR DESCRIPTION
**Pull request by jrconlin**
While moving to UUIDs is noble, they're not fully JSON encodable.
Normalize ChannelIDs to strings where possible.

closes #691

### CODE REVIEW

File: autopush/web/message.py [hunk 1]
```diff
@@ -9,7 +10,7 @@
 
 class MessageSchema(Schema):
     uaid = fields.UUID()
-    channel_id = fields.UUID(allow_none=True)
+    channel_id = fields.Str(allow_none=True)
```
Comment (bbangert) [inline autopush/web/message.py]: Changing this to str removes the validation that occurs when requiring it to be a valid UUID.

Comment (bbangert) [inline autopush/web/message.py]: If you really want it to be a str on the way out, the post_load should transform it after it's validated as UUID or None.

File: autopush/web/message.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (pjenvey) [inline autopush/web/message.py]: kill whitespace

File: autopush/web/message.py [hunk 3]
```diff
@@ -32,8 +33,8 @@ def extract_data(self, req):
         except (InvalidToken, InvalidTokenException):
             raise InvalidRequest("Invalid message ID",
                                  status_code=400)
-        return dict(uaid=notif.uaid,
-                    channel_id=notif.channel_id,
+        return dict(uaid=str(notif.uaid),
```
Comment (pjenvey) [inline autopush/web/message.py]: you can actually pass these through, marshmallow's UUID field should be happy taking UUID objects

File: autopush/web/message.py (hunk 2 above)
Comment (jrconlin) [inline autopush/web/message.py]: part of me prefers that, if nothing else than to show that there are no system includes in this file (and it even passes flake8 ;) ) Killing it anyways.

Comment (jrconlin) [inline autopush/web/message.py]: ✅

**Comment by goinnn** [comment]
@jrconlin @pjenvey Related: https://github.com/mozilla-services/autopush/issues/697

If you want I can test it

**Pull request merged**